ethereum/contracts/SwapCreator.sol

```diff
@@ -242,7 +258,7 @@ contract SwapCreator is ERC2771Context, Secp256k1 {
         }
     }
 
-    function verifySecret(bytes32 _s, bytes32 pubKey) internal pure {
-        if (!mulVerify(uint256(_s), uint256(pubKey))) revert InvalidSecret();
+    function verifySecret(bytes32 _s, bytes32 _hashedPubkey) internal pure {
```

Comment:
Would it be worth converting the documentation in this contract to NatSpec and ensuring that all functions and their parameters are documented?
https://docs.soliditylang.org/en/v0.8.19/style-guide.html#natspec

Reply:
yeah probably, can do in a follow up